node-tests/fixtures/test-helper-with-load.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable ember/no-test-support-import */
```

Comment:
latest `eslint-plugin-ember` complaints about import from `test-support` but this is indeed test related code